dist/index.js

```diff
@@ -76,16 +76,16 @@ function movePositionByLines(code, numLines, position) {
     }
     if (opts.trimNewline) position++; // don't include prior newline
   } else if (numLines > 0) {
-      var numFollowingLines = numLines;
-      position++;
-      while (position < code.length && numFollowingLines > 0) {
-        if (code[position] === '\n') {
-          numFollowingLines--;
-        }
-        position++;
```

Comment:
I see you moved this `position++` outside of the block - did I make a fencepost error here?